### PR TITLE
Creation of Flask App

### PR DESCRIPTION
**Pull request by Bsubs**
Created the skeleton structure with a consistent theme with a navbar which allows for easy access to all pages with a shallow tree structure. Also implemented the homepage with large buttons as discussed in the meeting.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,4 +1,114 @@
...
+}
+
+/* .scroll_text {
+    text-align: center;
+    position: relative;
+    top: 20%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: royalblue;
+} */
```
Comment (jwyatt1999) [inline style.css]: Is this meant to be removed?

File: style.css [hunk 2]
```diff
@@ -1,4 +1,114 @@
...
+
+
+/* .info_header, .rule_header {
+    text-align: center;
+    position: relative;
+    top: 20%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: royalblue;
+} */
```
Comment (jwyatt1999) [inline style.css]: Is this also meant to be removed?

**Review by jwyatt1999 — CHANGES_REQUESTED** [review]
I think the changes look fine. The skeleton pages will definitely need some attention but we will get to that at some point and these are fine starting points. Just had a few queries regarding some commented css. 

Also, we aren't able to merge your PR just yet because there is a merge conflict (I pulled from main and you pulled from main, I then edited styles.css in my PR and have merged my PR, and you've made some changes to styles.css in your PR so github doesn't know what the original styles.css should be changed to). This can be resolved by a merge conflict resolution program after pulling in my commits to your branch, though I don't know if VSCode naturally has one (it certainly could) - I would err on the side of caution and resolve the merge conflict in github by clicking the 'Web Editor' button at the bottom of the PR near where it says 'Resolve conflicts'. I can help you with this if you'd like, it looks like we just need to add the two files together since we don't have any common id's/classes we are both applying styles to :)

**Comment by jwyatt1999** [comment]
Also, have you run the main page and styles.css through the HTML and CSS validator?

File: cartographers/static/style.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
+/* CSS for Background */
```
Comment (jwyatt1999) [inline cartographers/static/style.css:1]: Has this page been put through the CSS validator?

File: cartographers/templates/game_page.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    <td id="playerPoints">Points: 0</td>
+                    <td id="playerCoins">Coins: 0</td>
+                    <td id="pieceCoin">You will gain a coin from this piece:</td>
+                </tr>
+                <!-- <tr>
+                    <td id="playerCoins">Coins: 0</td>
+                </tr>
+                <tr>
+                    <td id="pieceCoin">You will gain a coin from this piece:</td>
+                </tr> -->
```
Comment (jwyatt1999) [inline cartographers/templates/game_page.html]: This commented section should be deleted

**Review by jwyatt1999 — COMMENTED** [review]
Just found a comment that should probably be removed. I'll be happy to approve this once the css file (...\static\style.css) and html files (all of those in ...\templates) have been run through the validator

File: cartographers/templates/game_page.html (hunk 4 above)
Comment (Bsubs) [reply cartographers/templates/game_page.html]: Removed the unecessary comment

**Comment by jwyatt1999** [comment]
This looks good to me!

**Review by jwyatt1999 — APPROVED** [review]

**Pull request merged**